package.json

```diff
@@ -23,7 +23,7 @@
     "@mate-academy/bemlint": "^0.1.1",
     "@mate-academy/eslint-config": "*",
     "@mate-academy/linthtml-config": "0.0.2",
-    "@mate-academy/scripts": "^0.2.0",
+    "@mate-academy/scripts": "^1.2.8",
```

Comment:
just curiosity why did you change scripts ?

Reply:
i didn't change, i think during the npm install it is automatically changed.